AnnoyancesFilter/Other/sections/self-promo.txt

```diff
@@ -187,6 +187,7 @@ www.kinopoisk.ru##footer > div:has(> div > div[data-testid="kp-ui.amediateka-foo
 !
 ! SECTION: Self-promo - Regular rules
 !
+washingtontimes.com##.article-text > div:has(a[href^="https://google.com/preferences/source?q="])
```

Comment:
```suggestion
washingtontimes.com##.article-text > div:has(> a[href^="https://google.com/preferences/source?q="])
```
Missed the newsletter. You can add the domain to this rule.
```adblock
tlz.de##.piano-newsletter-inline
```
There was a leftover from the popup, but I've fixed it.